cloudify/event.py

```diff
@@ -75,7 +75,12 @@ def operation_info(self):
 
     @property
     def text(self):
-        message = self._event['message']['text'].encode('utf-8')
+        message = self._event['message']['text']
+        try:
+            message = message.encode('utf-8')
+        except UnicodeDecodeError as ex:
+            message = "<Unable to encode to UTF-8; error mesage: {0}>".format(
+                str(ex))
```

Comment:
no need for the `str()` call, using `.format` to format it into a string already does that

Reply:
also this should be UnicodeEncodeError, which is what .encode throws, no?

Reply:
@tehasdf I thought that too, but it actually raised `UnicodeDecodeError`